fn_ioc_parser/fn_ioc_parser/util/customize.py

```diff
@@ -1,99 +1,393 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
 # -*- coding: utf-8 -*-
 
 """Generate the Resilient customizations required for fn_ioc_parser"""
 
 from __future__ import print_function
 from resilient_circuits.util import *
 
+def codegen_reload_data():
+    """Parameters to codegen used to generate the fn_ioc_parser package"""
+    reload_params = {"package": u"fn_ioc_parser",
+                    "incident_fields": [], 
+                    "action_fields": [], 
+                    "function_params": [u"ioc_parser_artifact_id", u"ioc_parser_artifact_type", u"ioc_parser_artifact_value", u"ioc_parser_attachment_id", u"ioc_parser_incident_id", u"ioc_parser_task_id"], 
+                    "datatables": [], 
+                    "message_destinations": [u"fn_ioc_parser"], 
+                    "functions": [u"function_ioc_parser"], 
+                    "phases": [], 
+                    "automatic_tasks": [], 
+                    "scripts": [], 
+                    "workflows": [u"example_parse_iocs_artifact", u"example_parse_iocs_attachment"], 
+                    "actions": [u"Example: Parse IOCs - Artifact", u"Example: Parse IOCs - Attachment"] 
+                    }
+    return reload_params
+
 
 def customization_data(client=None):
     """Produce any customization definitions (types, fields, message destinations, etc)
        that should be installed by `resilient-circuits customize`
     """
 
     # This import data contains:
-    #   Function inputs: incidentId, inputType, artifactId
-    #   Message Destinations: iocpdest
-    #   Functions: ioc_parser
+    #   Function inputs:
+    #     ioc_parser_artifact_id
+    #     ioc_parser_artifact_type
+    #     ioc_parser_artifact_value
+    #     ioc_parser_attachment_id
+    #     ioc_parser_incident_id
+    #     ioc_parser_task_id
+    #   Message Destinations:
+    #     fn_ioc_parser
+    #   Functions:
+    #     function_ioc_parser
```

Comment:
* @MarkScherfling can you confirm if we should change the function name here?
* This change will cause a breaking change to ioc_parser v1.0.0